modules/modules.go

```diff
@@ -358,6 +400,8 @@ func initNewModule(name string, prep, start, stop func() error, dependencies ...
 		depNames:            dependencies,
 	}
 
+	newModule.Sleep(false)
```

Comment:
The new module should be created with sleepMode=false so that this actually does something.